Assets/UPM/Console/Command.cs

```diff
@@ -4,9 +4,13 @@
 using System.Reflection;
 using UnityEngine;
 
-
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = true, Inherited = false)]
 public class CommandAttribute : System.Attribute
 {
+    public string CustomName = ""; // maybe we wanna make a command named something that already exists, we can just add a custom name and use that instead :)
+                              // like this [Command(CustomName = "NameMe")]
+
+    public bool OnlyIngame = false; // maybe we make it so commands marked with this can only be run while the game is playing
```

Comment:
I was thinking of this as commands for "Everywhere" and commands for "The Editor" I think your addition of a third "In Game" makes sense. Sort of a Venn Diagram of command capabilities. We'll have to figure out what the default is later. I will probably change this to an Enum in a quick patch after the PR.